guides/common/modules/proc_disabling-dns-dhcp-tftp-for-unmanaged-networks.adoc

```diff
@@ -2,11 +2,15 @@
 = Disabling DNS, DHCP, and TFTP for unmanaged networks
 
 If you want to manage TFTP, DHCP, and DNS services manually, you must prevent {Project} from maintaining these services on the operating system and disable orchestration to avoid DHCP and DNS validation errors.
-However, {Project} does not remove the back-end services on the operating system.
++
+[IMPORTANT]
+====
+Disabling these proxy features means {Project} will no longer orchestrate DNS, DHCP, and TFTP, but it does stop or remove the corresponding services.
```

Comment:
I find the wording confusing. Can you please tell me what this means? Do you mean Proxy Server or network proxy?

Reply:
There was a "not" missing -- fixed, this should be clear now.